daemon/loop.go

```diff
@@ -149,6 +150,52 @@ func (d *Daemon) pullAndSync(logger log.Logger) {
 		}); err != nil {
 			logger.Log("err", err)
 		}
+
+		// Find notes in revisions.
+		for i, rev := range revisions {
+			// Skip last revision (since we requested HEAD~1 above)
+			if i == len(revisions)-1 {
+				continue
+			}
+			n, err := working.GetNote(rev)
+			if err != nil {
+				logger.Log("err", errors.Wrap(err, "loading notes from repo; possibly no notes"))
```

Comment:
Yes, usually the error is that there are no notes.

I think it's better to just log this error and move on, rather than producing a proper error, because this is quite normal when users first start using flux.